stable/console-chart/templates/subscription-fast.yaml

```diff
@@ -5,7 +5,7 @@ apiVersion: apps.open-cluster-management.io/v1
 kind: Subscription
 metadata:
   annotations:
-    apps.open-cluster-management.io/git-branch: release-2.2
+    apps.open-cluster-management.io/git-branch: main
```

Comment:
We should probably just use `release-2.5` here (because `main` fast forwards to `release-2.5` currently) and close the other PR against `release-2.5` branch.

Reply:
Done, Thank you